class.phpmailer.php

```diff
@@ -1815,9 +1816,27 @@ public function setLanguage($langcode = 'en', $lang_path = '')
             if (!self::isPermittedPath($lang_file) or !is_readable($lang_file)) {
                 $foundlang = false;
             } else {
-                // Overwrite language-specific strings.
-                // This way we'll never have missing translation keys.
-                $foundlang = include $lang_file;
+                //$foundlang = include $lang_file;
```

Comment:
I know it's only backport but do we need to keep disabled code?

Reply:
I'm in favour of 1:1 backports